tutorials/RecSys22tutorial/03_Customize_Merlin_Models.ipynb

```diff
@@ -36,7 +36,7 @@
    "id": "ebc9c8c6-9fab-435f-958b-923b0f3d6ff4",
```

Comment:
<p>why do we define it from scratch this time and do not use <code>dlrm_body = mm.DLRMBlock(schema, embedding_dim=64, bottom_block=mm.MLPBlock([128,64]), top_block= mm.MLPBlock([128, 64, 32]))</code></p><p>? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/publications/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='66'/>

Reply:
<p>good catch. I will fix it.</p>